app/config/parameters.yml.dist

```diff
@@ -54,3 +54,6 @@ parameters:
 
     tiqr_app_android_url: https://play.google.com/store/apps/details?id=org.tiqr.authenticator&hl=en
     tiqr_app_ios_url: https://itunes.apple.com/us/app/tiqr/id430838214?mt=8&ls=1
+
+    session_max_absolute_lifetime: 3600 # 1 hours * 60 minutes * 60 seconds
```

Comment:
I think the parameter names are clear enough for us, but maybe these parameters (and the configuration in general) could be documented a bit more (some comments in the dist would suffice, I think). I will make a tech debt issue for this.